public_html/TranslationMdwiki/mw.cx.TranslationMdwiki.js

```
@@ -0,0 +1,356 @@
+
+async function fetchSourcePageContent_mdwiki(page_title, targetLanguage, tr_type, user_name) {
+	mdwiki_last_url.url = "";
+	// ---
+	// if page_title start with "Video:" then tr_type = all
+	if (page_title.startsWith("Video:") || page_title.startsWith("video:")) {
+		tr_type = "all";
+	}
+	// ---
+	const result = await fetchSourcePageContent_mdwiki_new(page_title, targetLanguage, tr_type, user_name);
+	// ---
+	if (result && result.segmentedContent && targetLanguage === "sw") {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)